### ISSUE TITLE
Addon breaks storybook builds

### ISSUE THREAD

**Issue by TheSisb** [tuchk4/storybook-readme#7]
Followed the simple setup instructions and when I try to start storybook I am getting the following error:

```
ERROR in ./~/css-loader!./~/stylus-loader!./~/github-markdown-css/github-markdown.css
Module build failed: ParseError: /Users/thesisb/project/node_modules/github-markdown-css/github-markdown.css:3:22
   1| @font-face {
   2|   font-family: octicons-link;
   3|   src: url(data:font/woff;charset=utf-8;base64,d09GRgABAAAAAAZwABAAAAAACFQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABEU0lHAAAGaAAAAAgAAAAIAAAAAUdTVUIAAAZcAAAACgAAAAoAAQAAT1MvMgAAAyQAAABJAAAAYFYEU3RjbWFwAAADcAAAAEUAAACAAJThvmN2dCAAAATkAAAABAAAAAQAAAAAZnBnbQAAA7gAAACyAAABCUM+8IhnYXNwAAAGTAAAABAAAAAQABoAI2dseWYAAAFsAAABPAAAAZwcEq9taGVhZAAAAsgAAAA0AAAANgh4a91oaGVhAAADCAAAABoAAAAkCA8DRGhtdHgAAAL8AAAADAAAAAwGAACfbG9jYQAAAsAAAAAIAAAACABiATBtYXhwAAACqAAAABgAAAAgAA8ASm5hbWUAAAToAAABQgAAAlXu73sOcG9zdAAABiwAAAAeAAAAME3QpOBwcmVwAAAEbAAAAHYAAAB/aFGpk3jaTY6xa8JAGMW/O62BDi0tJLYQincXEypYIiGJjSgHniQ6umTsUEyLm5BV6NDBP8Tpts6F0v+k/0an2i+itHDw3v2+9+DBKTzsJNnWJNTgHEy4BgG3EMI9DCEDOGEXzDADU5hBKMIgNPZqoD3SilVaXZCER3/I7AtxEJLtzzuZfI+VVkprxTlXShWKb3TBecG11rwoNlmmn1P2WYcJczl32etSpKnziC7lQyWe1smVPy/Lt7Kc+0vWY/gAgIIEqAN9we0pwKXreiMasxvabDQMM4riO+qxM2ogwDGOZTXxwxDiycQIcoYFBLj5K3EIaSctAq2kTYiw+ymhce7vwM9jSqO8JyVd5RH9gyTt2+J/yUmYlIR0s04n6+7Vm1ozezUeLEaUjhaDSuXHwVRgvLJn1tQ7xiuVv/ocTRF42mNgZGBgYGbwZOBiAAFGJBIMAAizAFoAAABiAGIAznjaY2BkYGAA4in8zwXi+W2+MjCzMIDApSwvXzC97Z4Ig8N/BxYGZgcgl52BCSQKAA3jCV8CAABfAAAAAAQAAEB42mNgZGBg4f3vACQZQABIMjKgAmYAKEgBXgAAeNpjYGY6wTiBgZWBg2kmUxoDA4MPhGZMYzBi1AHygVLYQUCaawqDA4PChxhmh/8ODDEsvAwHgMKMIDnGL0x7gJQCAwMAJd4MFwAAAHjaY2BgYGaA4DAGRgYQkAHyGMF8NgYrIM3JIAGVYYDT+AEjAwuDFpBmA9KMDEwMCh9i/v8H8sH0/4dQc1iAmAkALaUKLgAAAHjaTY9LDsIgEIbtgqHUPpDi3gPoBVyRTmTddOmqTXThEXqrob2gQ1FjwpDvfwCBdmdXC5AVKFu3e5MfNFJ29KTQT48Ob9/lqYwOGZxeUelN2U2R6+cArgtCJpauW7UQBqnFkUsjAY/kOU1cP+DAgvxwn1chZDwUbd6CFimGXwzwF6tPbFIcjEl+vvmM/byA48e6tWrKArm4ZJlCbdsrxksL1AwWn/yBSJKpYbq8AXaaTb8AAHja28jAwOC00ZrBeQNDQOWO//sdBBgYGRiYWYAEELEwMTE4uzo5Zzo5b2BxdnFOcALxNjA6b2ByTswC8jYwg0VlNuoCTWAMqNzMzsoK1rEhNqByEyerg5PMJlYuVueETKcd/89uBpnpvIEVomeHLoMsAAe1Id4AAAAAAAB42oWQT07CQBTGv0JBhagk7HQzKxca2sJCE1hDt4QF+9JOS0nbaaYDCQfwCJ7Au3AHj+LO13FMmm6cl7785vven0kBjHCBhfpYuNa5Ph1c0e2Xu3jEvWG7UdPDLZ4N92nOm+EBXuAbHmIMSRMs+4aUEd4Nd3CHD8NdvOLTsA2GL8M9PODbcL+hD7C1xoaHeLJSEao0FEW14ckxC+TU8TxvsY6X0eLPmRhry2WVioLpkrbp84LLQPGI7c6sOiUzpWIWS5GzlSgUzzLBSikOPFTOXqly7rqx0Z1Q5BAIoZBSFihQYQOOBEdkCOgXTOHA07HAGjGWiIjaPZNW13/+lm6S9FT7rLHFJ6fQbkATOG1j2OFMucKJJsxIVfQORl+9Jyda6Sl1dUYhSCm1dyClfoeDve4qMYdLEbfqHf3O/AdDumsjAAB42mNgYoAAZQYjBmyAGYQZmdhL8zLdDEydARfoAqIAAAABAAMABwAKABMAB///AA8AAQAAAAAAAAAAAAAAAAABAAAAAA==) format('woff');
---------------------------^
   4| }
   5|
   6| .markdown-body {

expected ")", got ";"
```

I tried excluding `storybook-readme` from my webpack's css processing to no avail:

```
  {
    test: /\.(styl|css)?$/,
    exclude: ['storybook-readme', 'github-markdown-css'],
    loader: 'style!css!stylus',
    include: path.resolve(__dirname, '../')
  },
```

Any help is appreciated.

**Comment by tuchk4** [comment]
@TheSisb just try

```js
loaders: [{
  test: /highlight\.js\/styles.+\.css$/,
  loader: 'style!css',
}, {
  test: /\.css$/,
  exclude: [
    /highlight\.js\/styles.+\.css$/,
    /github\-markdown\-css\/.+\.css$/,
  ],
  loader: 'style!css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss',
}],
```

If this help - I  will think how to make it easier to setup

**Comment by TheSisb** [comment]
Yup, that worked. Thanks.

**Comment by ro-savage** [comment]
@tuchk4 Ran into the exact same problem today, and tried the same things as TheSisb.

Maybe we should update the readme, as following the readme for CSSModules didn't work (for me at least).

**Comment by tuchk4** [comment]
@ro-savage  Nearest plans - Make addon work correctly without additional webpack configuration.

> and tried the same things as TheSisb.

Is it worked for you?

**Comment by ro-savage** [comment]
The regex you provided worked.

But first thing I tried was
` exclude: ['storybook-readme', 'github-markdown-css'],`
and the a plain css loader with `include: ['storybook-readme', 'github-markdown-css']`.

As that seems to be what the readme was saying. However they didnt work.

**Comment by tuchk4** [comment]
I have published `storybook-readme@2.0.0` without additional webpack configuration. Only for `*.md` files

```js
module.exports = {
  module: {
    loaders: [{
      test: /\.md$/,
      loader: "raw"
    }]
  }
};
```
